inst/makeData.R

```diff
@@ -83,5 +83,26 @@ d = load2("../thirdparty/yeast.RData")
 yeast.task = makeMultilabelTask("yeast-example", data = d, target = paste0("label", 1:14))
 save(yeast.task, file = file.path(dn, "yeast.task.RData"), compress = "xz")
 
+# FDA classification
+set.seed(DATASEED)
+gunpoint = load2("../thirdparty/gunpoint.RData")
+gunpoint.task = makeFDAClassifTask(data = gunpoint, target = "X1", positive = "1")
+save(gunpoint.task, file = file.path(dn, "gunpoint.task.RData"), compress = "xz")
 
-
+# FDA regression
+set.seed(DATASEED)
+data(fuelSubset, package = "FDboost")
+fuelSubset$UVVIS <- scale(fuelSubset$UVVIS, scale = FALSE)
+fuelSubset$NIR <- scale(fuelSubset$NIR, scale = FALSE)
+fuelSubset$uvvis.lambda <- with(fuelSubset, (uvvis.lambda - min(uvvis.lambda)) /
+                                  (max(uvvis.lambda) - min(uvvis.lambda) ))
+fuelSubset$nir.lambda <- with(fuelSubset, (nir.lambda - min(nir.lambda)) /
+                                (max(nir.lambda) - min(nir.lambda) ))
+LEN1 = length(fuelSubset$uvvis.lambda)
+LEN2 = length(fuelSubset$nir.lambda)
+mdata = as.data.frame(Reduce(cbind, list(fuelSubset$UVVIS, fuelSubset$NIR, fuelSubset$h2o, fuelSubset$heatan)))
```

Comment:
Not doing it for now, as it might result in errors later on. @smilesun will fix this on friday.